lldb/include/lldb/ValueObject/DILParser.h

```diff
@@ -116,8 +116,10 @@ class DILParser {
 
   lldb::DynamicValueType m_use_dynamic;
   bool m_use_synthetic;
-  bool m_fragile_ivar;
-  bool m_check_ptr_vs_member;
+  // The following are not currently used, but will be used as more
+  // functionality is added to DIL.
+  bool m_fragile_ivar __attribute__((unused));
+  bool m_check_ptr_vs_member __attribute__((unused));
```

Comment:
This should be `LLVM_ATTRIBUTE_UNUSED`, but I'd rather just delete these. (one reason is that I want to see how they're actually being used) I believe the warning doesn't fire on unused function (constructor) arguments, so you can keep them there, instead of deleting all the plumbing.

Reply:
Done.